src/components/loginform/LoginForm.scss

```diff
@@ -186,4 +186,72 @@ opacity: 1;
     padding-bottom: 10px;
     }
 
+    @media only screen and (max-width: 767px) {
+      .login-container {
+        flex-direction: column;
+        background-image: url('../../images/loginform/Backgroundlogin.svg');
+        background-repeat: no-repeat;
+        background-size: cover;
+      }
+      .logotype-mobile {
+        margin-top: 50px;
+        margin-bottom: 20px; 
+        width: 300px;
+        height: 100px;
+      }
+      .login-right {
+        max-width: 400px;
+        width: 100%;
+      }
+      .login-buttons {
+        width: 100%;
+        display: flex;
+        justify-content: center;
+      }
+      
+      .login-buttons button {
+        background: #1A3835;
```

Comment:
check if this color already exist in file with colors and reuse it. if not then create new scss variable and use it here

Reply:
You have example in this file in line 173